framework/src/org/checkerframework/framework/source/SourceChecker.java

```diff
@@ -1729,6 +1744,17 @@ public final String getOption(String name) {
         return getOption(name, null);
     }
 
+    /**
+     * Determines the boolean value of the option with the given name. Returns false if the option
+     * is not set.
+     *
+     * @see SourceChecker#getLintOption(String,boolean)
+     */
+    @Override
+    public final boolean getBooleanOption(String name) {
+        return Boolean.valueOf(getOption(name, "true"));
```

Comment:
The documentation says that `Returns false if the option is not set.`
However, here `true` is provided as the default value. So doesn't this return `true` if the option is not set?